docs/fundamentals/application-insights/sample/src/AppInsightsDemo/Startup.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Microsoft.AspNet.Authentication.Facebook;
+using Microsoft.AspNet.Authentication.MicrosoftAccount;
+using Microsoft.AspNet.Builder;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Diagnostics.Entity;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.Logging;
+using AppInsightsDemo.Models;
+using Microsoft.ApplicationInsights.AspNet;
```

Comment:
I don't think you need this namespace to be inserted any longer. If you do - sort using statements